### PR TITLE
Prepared support for Bicep 0.4.1

### PR DESCRIPTION
**Pull request by SimonWahlin**
Added new dependencies to download script and module manifest.

### CODE REVIEW

**Comment by SimonWahlin** [comment]
All tests are failing due to  assemblyversion.txt containing the old tag.

Once a new tag is created in the Bicep repository we can update assemblyversion.txt in this PR and tests should run.

File: scripts/downloadDependencies.ps1 [hunk 1]
```diff
@@ -36,13 +36,18 @@ try {
...
-        'Azure.Bicep.Types.dll',
         'Azure.Bicep.Types.Az.dll',
+        'Azure.Bicep.Types.dll',
         'Azure.Deployments.Core.dll',
-        'Azure.Deployments.Expression.dll', 
+        'Azure.Deployments.Expression.dll',
         'Bicep.Core.dll',
         'Bicep.Decompiler.dll',
-        'Newtonsoft.Json.dll'
+        'BicepTypes.json',
```
Comment (StefanIvemo) [inline scripts/downloadDependencies.ps1:45]: `BicepTypes.json` should be removed from here

**Comment by StefanIvemo** [comment]
Can you update the assemblyversion.txt to v0.4.1, and make a new commit?

**Comment by SimonWahlin** [comment]
This should fix #163

**Comment by StefanIvemo** [comment]
Need to update this one again, new version with fixes released, v0.4.63.

**Review by StefanIvemo — DISMISSED** [review]

**Review by StefanIvemo — APPROVED** [review]

**Pull request merged**